src/store/src/Bridge/ChromaDb/Store.php

```diff
@@ -53,24 +53,41 @@ public function add(VectorDocument ...$documents): void
     }
 
     /**
-     * @param array{where?: array<string, string>, whereDocument?: array<string, mixed>} $options
+     * @param array{where?: array<string, string>, whereDocument?: array<string, mixed>, include?: array<string>} $options
```

Comment:
which expectations do we have here for the `include` option?

if i just want `embeddings`, `metadatas` and `distances` - how would i enable that?

Reply:
If you don't want to specify anything, simply leave it out. Then the default from the ChromaDB extension will be used. That's how it was before. However, as soon as something is passed by the user, we should ensure that at least the metadata and embeddings are included, which the array merge does (maybe we can remove distance)

Reply:
> If you don't want to specify anything, simply leave it out.

yup, got that, that's good :+1: 

i understand that we basically treat `['embeddings', 'metadatas', 'distances']` as the default here, right?
what would i need to do, to enable that default? i'm questioning if that even works with that current implementation. adding a test here would be great

Reply:
Thanks for reviewing the code and for the comments.

I've added some tests.
The default `$include = null` should work since CodeWithKyrian/chromadb-php has the correct default (see link).

https://github.com/CodeWithKyrian/chromadb-php/blob/3bf9f8db6ebfbd805774fa81176f5029afab0371/src/Resources/CollectionResource.php#L279

However, we can also define our own default if you like...